tests/beta/test_default_tensor_tf.py

```diff
@@ -505,7 +505,7 @@ def circuit(x):
         )
         assert np.allclose(res, expected, atol=tol, rtol=0)
 
-    @pytest.mark.parametrize("diff_method", ALLOWED_DIFF_METHODS)
+    @pytest.mark.parametrize("diff_method", ["parameter-shift", "finite-diff", "device"])
```

Comment:
Was this changed so that tests were more explicit (not depending on something that is defined in code)?

I think then `ALLOWED_DIFF_METHODS` is unused, I'm sure the linter will be complaining :smile:

Reply:
With #552, there was an extension:
`ALLOWED_DIFF_METHODS = ("best", "classical", "device", "parameter-shift", "finite-diff")`

In the ``"classical"`` case (with `PassthruQNode`) there is not `jacobian` method available, it needs to be excluded. Also, `"best"` will result in ``"classical"`` for `default.tensor.tf`.

Reply:
:ok: